test/test_belief_propagation.jl

```diff
@@ -21,149 +20,149 @@ using ITensorNetworks:
-  exact_sz = contract_inner(Oψ, ψ) / contract_inner(ψ, ψ)
+  # Oψ = copy(ψ)
+  # Oψ[v] = apply(op("Sz", s[v]), ψ[v])
+  # exact_sz = inner(Oψ, ψ) / inner(ψ, ψ)
 
-  bpc = BeliefPropagationCache(ψψ, group(v -> v[1], vertices(ψψ)))
-  bpc = update(bpc)
-  env_tensors = environment(bpc, [PartitionVertex(v)])
-  numerator = contract(vcat(env_tensors, ITensor[ψ[v], op("Sz", s[v]), dag(prime(ψ[v]))]))[]
-  denominator = contract(vcat(env_tensors, ITensor[ψ[v], op("I", s[v]), dag(prime(ψ[v]))]))[]
+  # bpc = BeliefPropagationCache(ψψ, group(v -> v[1], vertices(ψψ)))
+  # bpc = update(bpc)
+  # env_tensors = environment(bpc, [PartitionVertex(v)])
+  # numerator = contract(vcat(env_tensors, ITensor[ψ[v], op("Sz", s[v]), dag(prime(ψ[v]))]))[]
+  # denominator = contract(vcat(env_tensors, ITensor[ψ[v], op("I", s[v]), dag(prime(ψ[v]))]))[]
 
-  @test abs.((numerator / denominator) - exact_sz) <= 1e-14
+  # @test abs.((numerator / denominator) - exact_sz) <= 1e-14
 
-  #Test updating the underlying tensornetwork in the cache
-  v = first(vertices(ψψ))
-  new_tensor = randomITensor(inds(ψψ[v]))
-  bpc = update_factor(bpc, v, new_tensor)
-  ψψ_updated = tensornetwork(bpc)
-  @test ψψ_updated[v] == new_tensor
+  # #Test updating the underlying tensornetwork in the cache
+  # v = first(vertices(ψψ))
+  # new_tensor = randomITensor(inds(ψψ[v]))
+  # bpc = update_factor(bpc, v, new_tensor)
+  # ψψ_updated = tensornetwork(bpc)
+  # @test ψψ_updated[v] == new_tensor
 
-  #Now test on a tree, should also be exact
-  g = named_comb_tree((4, 4))
-  s = siteinds("S=1/2", g)
-  χ = 2
-  Random.seed!(1564)
-  ψ = random_tensornetwork(s; link_space=χ)
+  # #Now test on a tree, should also be exact
+  # g = named_comb_tree((4, 4))
+  # s = siteinds("S=1/2", g)
+  # χ = 2
+  # Random.seed!(1564)
+  # ψ = random_tensornetwork(s; link_space=χ)
 
-  ψψ = ψ ⊗ prime(dag(ψ); sites=[])
+  # ψψ = ψ ⊗ prime(dag(ψ); sites=[])
 
-  v = (1, 3)
+  # v = (1, 3)
 
-  Oψ = copy(ψ)
-  Oψ[v] = apply(op("Sz", s[v]), ψ[v])
-  exact_sz = contract_inner(Oψ, ψ) / contract_inner(ψ, ψ)
+  # Oψ = copy(ψ)
+  # Oψ[v] = apply(op("Sz", s[v]), ψ[v])
+  # exact_sz = inner(Oψ, ψ) / inner(ψ, ψ)
 
-  bpc = BeliefPropagationCache(ψψ, group(v -> v[1], vertices(ψψ)))
-  bpc = update(bpc)
-  env_tensors = environment(bpc, [PartitionVertex(v)])
-  numerator = contract(vcat(env_tensors, ITensor[ψ[v], op("Sz", s[v]), dag(prime(ψ[v]))]))[]
-  denominator = contract(vcat(env_tensors, ITensor[ψ[v], op("I", s[v]), dag(prime(ψ[v]))]))[]
+  # bpc = BeliefPropagationCache(ψψ, group(v -> v[1], vertices(ψψ)))
+  # bpc = update(bpc)
+  # env_tensors = environment(bpc, [PartitionVertex(v)])
+  # numerator = contract(vcat(env_tensors, ITensor[ψ[v], op("Sz", s[v]), dag(prime(ψ[v]))]))[]
+  # denominator = contract(vcat(env_tensors, ITensor[ψ[v], op("I", s[v]), dag(prime(ψ[v]))]))[]
 
-  @test abs.((numerator / denominator) - exact_sz) <= 1e-14
+  # @test abs.((numerator / denominator) - exact_sz) <= 1e-14
```

Comment:
I would rather not leave code commented out, we should delete it and bring it back later as needed. Same for down below.

Could you explain why it is being removed?

Reply:
Sorry yeah I had commented it out to focus on one part of the tests but forgot to change it back. Fixing that now.